service/src/main/java/bio/terra/cbas/initialization/InstanceInitializerBeanConfig.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.cbas.initialization;
+
+import bio.terra.cbas.config.CbasContextConfiguration;
+import bio.terra.cbas.dao.MethodDao;
+import bio.terra.cbas.dao.MethodVersionDao;
+import bio.terra.cbas.dao.RunSetDao;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class InstanceInitializerBeanConfig {
+
+  @Bean
+  public BackfillOriginalWorkspaceIdBean backfillOriginalWorkspaceIdBean(
```

Comment:
Oh (can you tell I'm putting this together as I walk through the code)... this is how the bean gets constructed.

Feel free to double check my terminology but I believe that _this_ is a bean, and the `InstanceInitializer` gets a bean, but the `BackfillOriginalWorkspaceIdBean` class is not necessarily anything bean-y.

So I guess it's up to you, my minor thought would be to rename the `BackfillOriginalWorkspaceIdBean` something more generic (as suggested above), but leave the named references to it as being called "`backfillOriginalWorkspaceIdBean`". What do you think?

Reply:
and I am responding as I read your comments, without perfect memory of what I wrote!

let me think about it a bit more carefully and get back to you